hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/DatanodeConfiguration.java

```diff
@@ -72,13 +72,25 @@
    */
   @Config(key = "replication.streams.limit",
       type = ConfigType.INT,
-      defaultValue = "10",
+      defaultValue = "12",
       tags = {DATANODE},
       description = "The maximum number of replication commands a single " +
           "datanode can execute simultaneously"
   )
   private int replicationMaxStreams = REPLICATION_MAX_STREAMS_DEFAULT;
 
+  /**
+   * The maximum number of replication requests.
+   */
+  @Config(key = "replication.queue.limit",
```

Comment:
Cany you please add this new config to `ReplicationServer#ReplicationConfig` instead of `DatanodeConfig`?  That is a more specific class for config related to replication.  If we also moved existing `replicationMaxStreams`, too, then we could avoid introducing `DatanodeConfiguration` in `ReplicationServer`?

Reply:
Moving existing config in #2943.

Reply:
Done in merge from `master`.